OutOfSchool/OutOfSchool.WebApi/Common/Exceptions/PictureNotFoundException.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace OutOfSchool.WebApi.Common.Exceptions
+{
+    [Serializable]
+    public class PictureNotFoundException : Exception
+    {
```

Comment:
maybe just use this https://docs.microsoft.com/en-us/dotnet/api/system.io.filenotfoundexception?view=netcore-3.1